packages/paste-core/components/chat-log/src/types.ts

```diff
@@ -10,3 +10,9 @@ export interface ChatBubbleProps {
   children?: React.ReactNode;
   element?: string;
 }
+
+export interface ChatMessageMetaProps {
+  ['aria-label']?: string;
+  children: React.ReactNode;
```

Comment:
Are there any types within `React.ReactNode` that would not be valid for this component? 
More specifically, can `children` be `null` or `undefined` for this component?

I see that on line 35 above we are using `isRequired`, which makes it seem like this should exclude those values.

Happy to chat about this synchronously if you have questions, this stuff can be tricky.